### DISCUSSION TITLE
Library Improvements

### DISCUSSION THREAD

**Discussion by samuelint** [samuelint/langchain-openai-api-bridge#36] [General]
Hi @mynameiskenlee,
I saw you forked this GitHub project.
If you find anything missing, feel free to open a pull request. Your contributions would be very appreciated!
Also, if you could give the project a star, it would mean a lot and help us out!
Thanks!

**Comment by mynameiskenlee** [comment]
Hi @samuelint,

We are intending to use your library in one of our project with Google Vertex AI, and we did find some bug that the chat id returning the same in every chat completion stream. I have just submitted a pull request to get it fixed.

**Reply by samuelint** [reply]
I've never tested the library with Vertex AI. I'd love to hear how the integration went.

Thanks for the pull request :)

**Reply by samuelint** [reply]
It's now deployed on the latest version
https://github.com/samuelint/langchain-openai-api-bridge/releases/tag/v0.11.2
